Lib/inspect.py

```diff
@@ -2495,6 +2477,46 @@ def _signature_from_function(cls, func, skip_bound_arg=True,
                __validate_parameters__=is_duck_function)
 
 
+def _signature_from_builtin(cls, func, skip_bound_arg=True):
```

Comment:
You may rename it to _signature_from_builtin_func() ;-)

Reply:
This is not actually something I change :)

This is git acting strange, because I just moved some code above. 

I would prefer not to touch these parts. They are unrelated. I try to move code again to change the diff.